packages/appstore/src/constants/routes-config.js

```diff
@@ -0,0 +1,31 @@
+import { localize } from '@deriv/translations';
```

Comment:
why changed ts file to js?

Reply:
Most of the routes-config files are .js, and have needed some efforts to migrate it to .ts, That is why kept it as .js.

Now changed to .ts

Reply:
yeah, the goal is to migrate to `ts` and `tsx`,so we should not move backward